test/acceptance/duplicate_tx_test.go

```diff
@@ -0,0 +1,85 @@
+package acceptance
+
+import (
+	"bytes"
+	"context"
+	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-network-go/test/harness"
+	"github.com/orbs-network/orbs-spec/types/go/protocol"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+func TestSendSameTransactionMoreThanOnce1(t *testing.T) {
+	harness.Network(t).AllowingErrors(
+		"error adding transaction to pending pool",
+		"error adding forwarded transaction to pending pool",
+		"error sending transaction",
+	).Start(func(ctx context.Context, network harness.TestNetworkDriver) {
+		ts := time.Now()
+
+		contract := network.GetBenchmarkTokenContract()
+		contract.DeployBenchmarkToken(ctx, 1)
+
+		// send three identical transactions to two nodes
+		network.SendTransactionInBackground(ctx, builders.TransferTransaction().WithTimestamp(ts).Builder(), 0)
+		response0 := <- network.SendTransaction(ctx, builders.TransferTransaction().WithTimestamp(ts).Builder(), 1)
+		response1 := <- network.SendTransaction(ctx, builders.TransferTransaction().WithTimestamp(ts).Builder(), 1)
+
+		// second transaction will be accepted into the pool and committed by the internode sync
```

Comment:
instead of comments, add messages to the assertion so that if they fail, the message appears in the log: "second transaction should be accepted into the pool and committed by the internode sync"

Reply:
done